openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -154,13 +157,20 @@ def set_logged_in_cookies(request, response, user):
     return response
 
 
-def refresh_jwt_cookies(request, response, user):
+def refresh_jwt_cookies(request, refresh_status, user):
```

Comment:
Now that this creates the response as well, would a name like `get_response_with_refreshed_jwt_cookies` be more clear?